SYCL/AOT/multiple-devices.cpp

```diff
@@ -18,11 +18,19 @@
 // RUN: %GPU_RUN_PLACEHOLDER %t_all.out
 // RUN: %ACC_RUN_PLACEHOLDER %t_all.out
 
+// FIXME: Change the behavior when proper automation for assert support is
+// introduced
```

Comment:
Will it be fixed by https://github.com/intel/llvm/pull/3767 ?

Reply:
Per offline discussion, agreed to elaborate on the comment in distinct patch.